JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCardYourNetwork.kt

```diff
@@ -26,60 +27,91 @@ import androidx.ui.layout.Container
 import androidx.ui.layout.LayoutHeight
 import androidx.ui.layout.LayoutPadding
 import androidx.ui.layout.LayoutSize
-import androidx.ui.material.EmphasisLevels
 import androidx.ui.material.MaterialTheme
 import androidx.ui.material.ProvideEmphasis
 import androidx.ui.material.ripple.Ripple
 import androidx.ui.material.surface.Card
 import androidx.ui.res.imageResource
 import androidx.ui.text.style.TextOverflow
+import androidx.ui.tooling.preview.Preview
 import androidx.ui.unit.dp
 import com.example.jetnews.R
+import com.example.jetnews.data.post1
 import com.example.jetnews.model.Post
+import com.example.jetnews.model.PostAuthor
 import com.example.jetnews.ui.Screen
 import com.example.jetnews.ui.navigateTo
 
+private val cardSize = LayoutSize(280.dp, 240.dp)
+
 @Composable
-fun PostCardPopular(post: Post) {
-    Card(shape = RoundedCornerShape(4.dp)) {
+fun PostCardPopular(post: Post, modifier: Modifier = Modifier.None) {
+    Card(shape = RoundedCornerShape(4.dp), modifier = modifier) {
```

Comment:
Nit: same can be applied for some function calls in this PR, I'd pass the modifier param first. Not big deal though

Reply:
+1